### DISCUSSION TITLE
[RFC] 067 - Pinned Assistant List 侧边栏常驻助手

### DISCUSSION THREAD

**Discussion by arvinxx** [lobehub/lobehub#4515] [RFCs]
## 背景

希望合理利用现有侧边栏，并提升高频助手的使用便捷度。

## 设计

暂时做简单一些，允许将现有的置顶助手显示在侧边栏。激活时将隐藏 session 部分，获得更大的对话区域。

目前暂未默认开启，如需体验，添加 `FEATURE_FLAGS=+pin_list` 开启

<img width="1720" alt="image" src="https://github.com/user-attachments/assets/6935e155-4a1d-4ab7-a61a-2b813d65bb7b">

<img width="1723" alt="image" src="https://github.com/user-attachments/assets/c68e88e4-cf2e-4122-82bc-89ba193b1eb4">

## 进展

- 快速实现，通过`FEATURE_FLAGS=+pin_list` 开启： #4514

**Comment by leter** [comment]
1.26 版本开启这个参数不起作用呀

**Reply by arvinxx** [reply]
怎么不起作用

**Comment by Wxh16144** [comment]
终于开始做快捷键有关功能了， 之前都是自己复制 URL 搭配 Raycast 的快捷键使用

**Comment by canisminor1990** [comment]
Features:

- [ ] 侧边栏 PIN 和 置顶逻辑分离，目前首屏比较重复而且排序不稳定
- [ ] 侧边栏支持拖拽，添加和移除
- [ ] 支持自定义功能入口不仅限助手 (长期)

Styles
- [x] 指示器可以参考 Discord, 目前描边的有点干扰 https://github.com/lobehub/lobe-chat/pull/5533
- [ ] 头像尺寸视觉上可以和上方按钮靠齐，现在看着有点大
